src/libs/portfolio/portfolio.ts

```diff
@@ -287,7 +295,15 @@ export class Portfolio {
       .filter(Boolean) as string[]
 
     // Remove duplicates and always add ZeroAddress
-    hints.erc20s = [...new Set(checksummedErc20Hints.concat(ZeroAddress))]
+    let filteredErc20s = [...new Set(checksummedErc20Hints.concat(ZeroAddress))]
+    // Exclude tokens by chainId/address
+    filteredErc20s = filteredErc20s.filter(
+      (addr) =>
+        !EXCLUDED_TOKENS.some(
+          (ex) =>
+            ex.chainId === this.network.chainId && ex.address.toLowerCase() === addr.toLowerCase()
```

Comment:
hints.erc20s are no longer deduplicated. If they are returned by the lib in the future they will be different than the ones that are actually used. Imo, modify hints.erc20s directly